app/src/main/java/com/gh4a/fragment/CommitFragment.java

```diff
@@ -191,19 +198,19 @@ protected void fillStats(List<CommitFile> files, List<CommitComment> comments) {
             final LinearLayout parent;
 
             switch (file.getStatus()) {
-                case "added":
+                case Constants.STATUS_ADDED:
```

Comment:
These should go to [CommitFile.java](https://github.com/slapperwan/gh4a/blob/master/github-api/src/main/java/org/eclipse/egit/github/core/CommitFile.java) instead.